libc::lutimes is not defined when building with --target=wasm32-unknown-emscripten

I'm building a wasm project that depends on filetime and it looks like libc::lutimes is not defined for `wasm32-unknown-emscripten` targets.

Build log:
```
$ cargo web build --target=wasm32-unknown-emscripten
   Compiling filetime v0.1.15
error[E0425]: cannot find value `lutimes` in module `libc`
  --> /home/justas/.cargo/registry/src/github.com-1ecc6299db9ec823/filetime-0.1.15/src/unix/utimes.rs:12:42
   |
12 |     super::utimes(p, atime, mtime, libc::lutimes)
   |                                          ^^^^^^^ did you mean `utimes`?
error: aborting due to previous error
For more information about this error, try `rustc --explain E0425`.
error: Could not compile `filetime`.
error: build failed
```

This might just be an issue with libc so feel free to close it and I'll reopen it in the libc repo.

--- a/src/unix/mod.rs
+++ b/src/unix/mod.rs
@@ -18,7 +18,9 @@ cfg_if! {
     } else if #[cfg(any(target_os = "android",
                         target_os = "solaris",
                         target_os = "netbsd",
-                        target_os = "openbsd"))] {
+                        target_os = "openbsd",
+                        target_os = "freebsd",
+                        target_os = "emscripten"))] {
         mod utimensat;
         pub use self::utimensat::*;
     } else {

